### PR TITLE
`2101` add EditBadge to Table

### LINKED ISSUE #2101: Missing "New/Updated" UI chip for affected roles in the Edit Roles view.
Missing "New/Updated" UI chip for affected roles in the Edit Roles view.

screenshot

<details><summary>Browser metadata</summary>

```
Path:      /roles/edit?dao=sep:0xe20221A8939F9373C091262CBEF9493aca196acD
Browser:   Firefox 127.0 on Mac OS 10.15
Viewport:  1680 x 912 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-2081.app.dev.decentdao.org%2Froles%2Fedit%3Fdao%3Dsep%3A0xe20221A8939F9373C091262CBEF9493aca196acD&os=OS%20X&os_version=Catalina&browser=Firefox&browser_version=127.0&resolution=1680x912&speed=1&start=true&ref=netlify-source)

</details>

[Open Deploy Preview](https://deploy-preview-2081.app.dev.decentdao.org/roles/edit?dao=sep:0xe20221A8939F9373C091262CBEF9493aca196acD) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=668b30cffc4a0d0008ff3a94&commentID=668c120427123c64ec6a5265&resolution=resolved)

_Originally posted by @nicolaus-sherrill in https://github.com/decentdao/decent-interface/issues/2081#issuecomment-2214589107_

### PR DESCRIPTION
**Pull request by Da-Colon**
Closes #2101 

localhost_3000_roles_edit_dao=sep_0xd6Cb633E3A9cDEbA8a7A5CA4F30436F10FEbd418(Desktop)

### CODE REVIEW

**Review by DarksightKellar — APPROVED** [review]
noice

File: src/components/pages/Roles/RolesTable.tsx [hunk 1]
```diff
@@ -277,6 +280,9 @@ export function RolesRowEdit({
       <MemberColumn wearerAddress={wearerAddress} />
       <PayrollColumn payrollData={payrollData} />
       <VestingColumn vestingData={vestingData} />
+      <Td w="10%">
```
Comment (adamgall) [inline src/components/pages/Roles/RolesTable.tsx:283]: Is this hardcoded 10% ok on smaller screens? Any way to just tell this column/cell for its width to be something like "fit content"?

**Review by adamgall — CHANGES_REQUESTED** [review]

File: src/components/pages/Roles/RolesTable.tsx (hunk 1 above)
Comment (Da-Colon) [reply src/components/pages/Roles/RolesTable.tsx:283]: I wish, but I don't think so. I tried a few things but using a hardcoded percentage worked the best. The table isn't viewable on Mobile and this value works all the way down until it switches.

**Review by adamgall — APPROVED** [review]

**Review by nicolaus-sherrill — APPROVED** [review]
Looks good to me~

**Pull request merged**